### DISCUSSION TITLE
convert organizations view from grid to list

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7437] [Q&A]
Hi,  hope everyone is well. 

I am trying to convert my organization page view from a grid view to a list view.   

I am struggling to work this out so any pointers would be appreciated. 

I am using a standard CKAN install of 2.9.  

thanks

**Comment by ghost** [comment]
any suggestions or help would be appreciated.

**Comment by wardi** [comment]
Try overriding the template that renders the grid:
https://github.com/ckan/ckan/blob/master/ckan/templates/organization/snippets/organization_list.html

**Comment by ghost** [comment]
Thanks Ian, will give it another crack.  :D
